tests/TestCase.php

```diff
@@ -130,7 +130,7 @@ protected function createDriver()
      *
      * @return string
      */
-    protected function mapRemoteFilePath($file)
+    protected static function mapRemoteFilePath($file)
```

Comment:
I had to make this static since the new usecase is from a static context..

Reply:
this is a BC break

Reply:
Which new use case ?

Reply:
OK, saw it. To me, we should avoid doing such remapping in the data provider (which might involve splitting that case to a dedicated test)

Reply:
In the ideal world, I would agree, but this is a small low-impact change to fix a problem.
Splitting the test can be easily done later (and switching to non-static).

The being said, I just realised this is the first time I'm seeing static data provider - usually I'm making them non-static. Could that also be a solution?

Reply:
PHPUnit has deprecated the usage of non-static providers in PHPUnit 9.6 and forbids them in PHPUnit 10.

Splitting the test is much easier than asking PHPUnit to rollback this change.

Reply:
I tried splitting the test, but I didn't like the amount of duplicate code it causes, so I'm remapping the file paths based on an if condition. Not ideal IMO, but somewhat cleaner than a duplicate test.